src/lab.rs

```diff
@@ -401,22 +401,22 @@ mod test {
     #[test]
     fn red() {
         let a = Lab::from(Rgb::new(1.0, 0.0, 0.0));
-        let b = Lab::new(53.23288 / 100.0, 80.09246 / 128.0, 67.2031 / 128.0);
-        assert_relative_eq!(a, b, epsilon = 0.0001);
+        let b = Lab::new(53.23288, 80.09246, 67.2031);
+        assert_relative_eq!(a, b, epsilon = 0.01);
```

Comment:
I seem to remember that I recalculated those numbers in my attempt at this, but we can leave it as it is. It's still checked in other tests, as you said.